src/DataExporter.ts

```diff
@@ -151,32 +157,34 @@ export class DataExporter {
     let ids: Id64Array = [];
 
     if (writeHeaders) {
-      const header: string[] = (options.calculateMassProperties) ? ["total_count", "volume", "volume_count", "area", "area_count", "length", "length_count"] : [];
-      const outHeader = this.makeHeader(header, statement, options.dropIdColumnFromResult ? options.idColumn : -1);
+      const header: string[] = (options.calculateMassProperties) ? ["volume", "volume_si", "area", "area_si", "length", "length_si"] : [];
```

Comment:
Let's go back to the old output with total count then counts for volume, area and length ... the 'si' or suitability index is more confusing for someone to understand and outputting the base numbers allows you to calculate the suitability index later